src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt

```diff
@@ -222,13 +221,12 @@ class ComCaseloadService(
           LicenceStatus.TIMED_OUT,
         )
       }
-      validLicenceStatus == true && !offender.nomisRecord.isBreachOfTopUpSupervision()
```

Comment:
Are we not checking **is breach of top up supervision case** anymore??

Reply:
Yes it's already done by EligibilityService before we get here

Reply:
oh you are directly calling **getIneligibilityReasons**.
Ok.